spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__

```diff
@@ -0,0 +1 @@
+Did you know that git does not support storing empty directories?
```

Comment:
This seems to be an issue with either the template or the template generation. However for now it should stay here.